rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java

```diff
@@ -109,6 +109,18 @@
     @Configurable(description = "Options for the java command launching the node on the remote hosts", sectionSelector = 3)
     protected String javaOptions;
 
+    @Configurable(description = "autoGenerated, useStartupScript or useNodeJarStartupScript. Please refer to the manual", sectionSelector = 1)
+    protected String deploymentMode = "autoGenerated";
+
+    @Configurable(description = "The URL from which the node.jar will be downloaded. Used only when useNodeJarStartupScript mode is selected", sectionSelector = 3)
+    protected String nodeJarUrl = InitScriptGenerator.createNodeJarUrl();
+
+    @Configurable(textArea = true, description = "Standard Startup script that will start the ProActive node agents. Used only when useStartupScript mode is selected", sectionSelector = 3)
+    protected String startupScriptStandard = InitScriptGenerator.getDefaultLinuxStandardCommand();
+
+    @Configurable(textArea = true, description = "Startup script that will download a node.jar and start ProActive Agent using it. Used only when useNodeJarStartupScript mode is selected", sectionSelector = 3)
```

Comment:
Startup script that will download a Jave Runtime Environment, a jar archive containing all ProActive libraries (node.jar) and start ProActive Agent using it. Used only when useNodeJarStartupScript mode is selected.